README.md

```diff
@@ -59,14 +59,25 @@ You can run the demo with Marblerun in standalone mode as follows:
     pf_crypt encrypt --input plain/saved_model.pb --output models/resnet50-v15-fp32/1/saved_model.pb --wrap-key model_key
     ```
 
-1. Set the model key in Marblerun's manifest
+1. Generate a user certificate and key.
     ```bash
-    cat tf-server-manifest.json | sed "s|YOUR_KEY_HERE|$(hexdump -ve '1/1 "%02x"' model_key)|g" > manifest.json
+    openssl req -x509 -newkey rsa:4096 -sha256 -days 3650 -nodes -keyout user_credentials.key -out user_credentials.crt
```

Comment:
Is `days` required? Otherwise, I would not set the expiration date.

Reply:
taken directly from here: https://www.marblerun.sh/docs/workflows/define-manifest/#manifestadmins
Since these values are not actually verified by the coordinator, i would agree to remove expiration date.

Reply:
I think default is 30 days. What do you mean by "not actually verified"?

Reply:
Tested to make sure this actually works:

Generate a certificate that expired in 2008, using faketime:
```bash
faketime '2008-12-24 08:15:42' openssl req -x509 -newkey rsa:2048 -sha256 -days 2 -nodes -keyout admin_private.key -out admin_certificate.crt -subj "/C=DE"
```

Features requiring user credentials still work:
```bash
$ marblerun manifest set manifest.json localhost:4433
Successfully verified coordinator, now uploading manifest
Manifest signature: 08d1aa5923c1fe2058d32a9f7b410d8f6a8ecd50aa5a587edd2a6723168b59ca
Manifest successfully set

$ marblerun manifest update update_manifest.json localhost:4433 -c admin_certificate.crt -k admin_private.key
Successfully verified coordinator, now uploading manifest
Manifest successfully updated

# also works using curl
$ curl --cacert marblerun.crt --cert admin_certificate.crt --key admin_private.key -w "%{http_code}" --data-binary @update_manifest.json https://localhost:4433/update
{"status":"success","data":null}
```

Reply:
Thanks! I cross-checked the TLS config:
https://github.com/edgelesssys/marblerun/blob/dbd385ff46ef22695f393b7064ffe86501b0be59/coordinator/core/core.go#L220
seems to be ok